docs/www/configuration.md

```diff
@@ -466,6 +466,14 @@ pandaproxy_client:
   scram_password: ""
 
 rpk:
+  # Add optional flags to run rpk with specific parameters.
+  # The available start flags are found in: /src/v/config/configuration.cc
```

Comment:
This isn't the case, sadly. `/src/v/config/configuration.cc` has the part of the configuration file definition. The specific start options are spread throughout **seastar's** codebase:
- https://github.com/scylladb/seastar/blob/2cd0a2928782b89c71d16bc2d2628d987a17e874/src/util/log.cc#L506
- https://github.com/scylladb/seastar/blob/717396647823aaa670e18e4a33606a9806f450a4/src/core/reactor.cc#L3343-L3386

I might be missing some. I don't know what the right approach would be here. I think we need to add a "redpanda" (actually seastar) flags docs page, and perhaps contribute it to seastar, since it seems they don't have one.